app/setup_handlers.go

```diff
@@ -1,34 +1,38 @@
 package app
 
 import (
-	tokenregistrymigrations "github.com/Sifchain/sifnode/x/tokenregistry/migrations"
-	tokenregistrytypes "github.com/Sifchain/sifnode/x/tokenregistry/types"
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/mint"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
 )
 
-const upgradeNameV095 = "0.9.5"
+const upgradeNameV096 = "0.9.6"
```

Comment:
we don't need to do this now, but shouldn't the const name be more generic, without the version variable in it? (ie,. upgradeNameVersion)

Reply:
Good idea @marshsif - we should maybe think about a better way to handle migrations etc....